gradle/test-intellij.versions.toml

```diff
@@ -1,2 +1,2 @@
 [versions]
-versionList = "IIC-222.3739.24,IIC-221.6008.8,IIC-213.7172.6"
+versionList = "IIC-223.8836.41,IIC-221.6008.13,IIC-213.7172.25"
```

Comment:
This should fix the build error here: https://github.com/bufbuild/intellij-buf/actions/runs/4405172893/jobs/7715722729